service/frontend/workflow_handler.go

```diff
@@ -467,6 +467,99 @@ func (wh *WorkflowHandler) unaliasedSearchAttributesFrom(
 	return sa, nil
 }
 
+func (wh *WorkflowHandler) ExecuteMultiOperation(
+	ctx context.Context,
+	request *workflowservice.ExecuteMultiOperationRequest,
+) (_ *workflowservice.ExecuteMultiOperationResponse, retError error) {
+	defer log.CapturePanic(wh.logger, &retError)
+
+	if request == nil {
+		return nil, errRequestNotSet
+	}
+
+	if !wh.config.EnableExecuteMultiOperation(request.Namespace) {
+		return nil, errMultiOperationAPINotAllowed
+	}
+
+	// TODO: validation
+
+	namespaceName := namespace.Name(request.GetNamespace())
+	namespaceID, err := wh.namespaceRegistry.GetNamespaceID(namespaceName)
+	if err != nil {
+		return nil, err
+	}
+
+	workflowId := request.Operations[0].GetStartWorkflow().WorkflowId
```

Comment:
I don't see a check for whether there are more than zero operations, so this can panic